### ISSUE TITLE
Agent inserts empty ID into heartbeat

### ISSUE THREAD

**Issue by amol4anoff** [subutai-io/agent#307]
I use different peers and templates and keep getting error -container has not connected within 50 sec.

<img width="1346" alt="irelandpeer" src="https://user-images.githubusercontent.com/33647491/35552558-8a0149e6-05be-11e8-8bb6-5aea29daf520.png">

[karaflogs2018-01-30 12_39_37.txt.zip](https://github.com/subutai-io/agent/files/1676643/karaflogs2018-01-30.12_39_37.txt.zip)

**Comment by Dilshat** [comment]
Console awaits containers to come up and connect to it for 50 seconds. This is default recommended by prev sysnet team. We can increase this timeout if needed. However this will not solve the issue, since peer can be under stress and containers starting can take long time or even fail.
Console needs to have them started before processing with the next steps in environment creation

**Comment by Dilshat** [comment]
Seems the issue is related to the template `taskapp`. @emli please check

**Comment by Dilshat** [comment]
Seems the container had empty ID for some time. Need to review  cache on agent side and prevent empty value insertions into it
```
2018-01-30 13:12:44,881 | DEBUG | ool-166-thread-3 | CommandProcessor                 | 187 - io.subutai.core.executor.impl - 6.3.5.SNAPSHOT | Sending:
{
  "request": {
    "type": "EXECUTE_REQUEST",
    "id": "EB2BC3744A62E08D38C5DAAF9604FCFFF73472CC",
    "commandId": "acb50265-e52f-4b3c-84d2-a14dcb7fd455",
    "workingDirectory": "/",
    "command": "subutai clone id:fa72bb8b-570d-4aac-91c1-d498ff2fbe39 mysql-oJR-lgv-5-3 -i \"172.31.205.3/24 5\" -e 9d5253d4-01b0-4aec-9f55-1c87a0063d41 -t c577c48c-4022-4f2d-b50f-39a8f65b62ee \u0026\u0026 subutai hostname mysql-oJR-lgv-5-3 mysql-oJR",
    "stdOut": "RETURN",
    "stdErr": "RETURN",
    "runAs": "root",
    "timeout": 300,
    "isDaemon": 0
  }
}

...
   2018-01-30 13:13:04,821 | DEBUG | p1694229027-1969 | CommandProcessor                 | 187 - io.subutai.core.executor.impl - 6.3.5.SNAPSHOT | Response: ResponseImpl[type=EXECUTE_RESPONSE,id=EB2BC3744A62E08D38C5DAAF9604FCFFF73472CC,commandId=acb50265-e52f-4b3c-84d2-a14dcb7fd455,pid=0,responseNumber=2,stdOut=time="2018-01-30 13:12:59" level=info msg="mysql-oJR-lgv-5-3 started"
time="2018-01-30 13:13:00" level=info msg="mysql-oJR-lgv-5-3 with ID D6A7D1973F372671D2E4F936CB9B60281C382160 successfully cloned"
,stdErr=<null>,exitCode=0]

...


2018-01-30 13:13:05,836 | DEBUG | p1694229027-1866 | CommandProcessor                 | 187 - io.subutai.core.executor.impl - 6.3.5.SNAPSHOT | Heartbeat:

      {
        "status": "RUNNING",
        "name": "mysql-oJR-lgv-5-3",
        "quota": {},
        "interfaces": [
          {
            "interfaceName": "eth0",
            "ip": "172.31.205.3"
          }
        ],
        "environmentId": "9d5253d4-01b0-4aec-9f55-1c87a0063d41",
        "vlan": 5,
        "id": "",
        "hostname": "mysql-oJR",
        "arch": "AMD64"
 
}

...

2018-01-30 13:54:17,557 | DEBUG | qtp1726302524-88 | CommandProcessor                 | 187 - io.subutai.core.executor.impl - 6.3.5.SNAPSHOT | Heartbeat:

        "status": "RUNNING",
        "name": "mysql-oJR-lgv-5-3",
        "quota": {},
        "interfaces": [
          {
            "interfaceName": "eth0",
            "ip": "172.31.205.3"
          }
        ],
        "environmentId": "9d5253d4-01b0-4aec-9f55-1c87a0063d41",
        "vlan": 5,
        "id": "D6A7D1973F372671D2E4F936CB9B60281C382160",
        "hostname": "mysql-oJR",
        "arch": "AMD64"
      },
 ```

**Comment by Dilshat** [comment]
Fixed in 6.4.2
